sdk/eventhub/ci.data.yml

```diff
@@ -9,7 +9,7 @@ trigger:
     - release/*
   paths:
     include:
-    - sdk/eventhub/ci.data.yml
+    - sdk/eventhub/*
```

Comment:
Is is not doing what you expect it might be doing. See https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#paths they don't support wildcards so this will essentially have the pipeline trigger for all file changes under the directly (including subdirectories) which is what we want to avoid.

Reply:
Sad panda. Removing.